tests/containers/image.pm

```diff
@@ -28,8 +28,8 @@ sub run {
     my $versions = get_var('CONTAINER_IMAGE_VERSIONS', get_required_var('VERSION'));
     for my $version (split(/,/, $versions)) {
         my $images_to_test;
-        # Get list of images from CONTAINER_IMAGES_TO_TEST or use the default
-        unless ($images_to_test = get_urls_from_var('CONTAINER_IMAGES_TO_TEST')) {
+        # Get single image from CONTAINER_IMAGES_TO_TEST or array from get_suse_container_urls()
+        unless ($images_to_test = get_container_url_from_var('CONTAINER_IMAGE_TO_TEST')) {
```

Comment:
Following the previous comment, this would turn into:
```
unless (get_container_image_to_test()) {
```

Reply:
No. This does not make sense. Let me please handle this in separate pull request and please close.

Reply:
Or let me rephrase it: It think that this condition can be avoided completely.